third-party/rspamd/src/main/java/org/apache/james/rspamd/module/RSpamDModule.java

```diff
@@ -49,4 +57,11 @@ protected void configure() {
     public AdditionalInformationDTOModule<? extends TaskExecutionDetails.AdditionalInformation, ? extends AdditionalInformationDTO> webAdminFeedSpamAdditionalInformation() {
         return FeedSpamToRSpamDTaskAdditionalInformationDTO.SERIALIZATION_MODULE;
     }
+
+    @Provides
+    @Singleton
+    public RSpamDHttpClient rSpamDHttpClient(PropertiesProvider propertiesProvider) throws ConfigurationException, FileNotFoundException {
+        RSpamDClientConfiguration rSpamDClientConfiguration = RSpamDClientConfiguration.from(propertiesProvider.getConfiguration("rspamd"));
+        return new RSpamDHttpClient(rSpamDClientConfiguration);
+    }
```

Comment:
Should we create bean from `RSpamDClientConfiguration` level? (we inject it into the client https://github.com/apache/james-project/blob/eb2178a9269851cf4d15eed98669faa172cedd61/third-party/rspamd/src/main/java/org/apache/james/rspamd/client/RSpamDHttpClient.java).

And we still can set `RSpamDHttpClient` instance to singleton if we want.